src/components/EditableTagField/__snapshots__/EditableTagField.stories.storyshot

```diff
@@ -111,7 +111,7 @@ exports[`storybook Storyshots Components/Editable Tags Field Editable Tag Field
         <button
           aria-disabled={false}
```

Comment:
😭 I don't know why it looks like this

<img width="503" alt="Screen Shot 2022-11-08 at 2 18 15 PM" src="https://user-images.githubusercontent.com/47064971/200656179-6e25208e-1e90-4bad-aac0-fd8ef67b52a7.png">